docs/docs/eslint.md

```diff
@@ -31,59 +29,155 @@ rm .prettierrc
 touch .eslintrc.js
 ```
 
-### Configuring ESLint
+## Configuring ESLint
 
-We recommend copying our default .eslintrc.js content below to your newly created `.eslintrc.js` file and modifying it per your needs. Reference ESLint's [rules documentation](https://eslint.org/docs/rules/) for more options.
+To get started configuring ESLint we recommend copying our default ESLint content below to your newly created `.eslintrc.js` file and modifying it per your needs. Reference ESLint's [rules documentation](https://eslint.org/docs/rules/) for more options. Sample rules from the plugins are also provided to demonstrate their capabilities.
 
 ```js:title=.eslintrc.js
 module.exports = {
-  extends: ["standard"],
-  plugins: ["standard", "react"],
+  extends: [
+    "plugin:react/recommended",
+    "plugin:jsx-a11y/recommended",
+    "prettier",
+    "prettier/react",
+  ],
+  plugins: ["prettier", "react", "jsx-a11y"],
+  parser: "babel-eslint",
+  parserOptions: {
+    ecmaVersion: 8,
+    ecmaFeatures: {
+      impliedStrict: true,
+      classes: true,
+      jsx: true,
+    },
+  },
+  env: {
+    browser: true,
+    es6: true,
+  },
+  settings: {
+    react: {
+      version: "latest",
+    },
+  },
   rules: {
-    "no-var": "error", // optional, recommended when using es6+
-    "no-unused-vars": 1, // recommended
-    "arrow-spacing": ["error", { before: true, after: true }], // recommended
-    indent: ["error", 2],
-    "comma-dangle": [
-      "error",
-      {
-        objects: "only-multiline",
-        arrays: "only-multiline",
-        imports: "never",
-        exports: "never",
-        functions: "never",
-      },
-    ],
+    // https://github.com/yannickcr/eslint-plugin-react#configuration
+    "react/jsx-filename-extension": ["warn", { extensions: [".js", ".jsx"] }],
+    "react/jsx-uses-react": "error",
+    "react/react-in-jsx-scope": "error",
+    "react/no-deprecated": "error",
+    "react/prefer-stateless-function": "warn",
 
-    // options to emulate prettier setup
-    semi: ["error", "never"],
-    "max-len": ["error", { code: 80 }],
-    "template-curly-spacing": ["error", "always"],
-    "arrow-parens": ["error", "as-needed"],
+    // https://www.npmjs.com/package/eslint-plugin-jsx-a11y#supported-rules
+    "jsx-a11y/accessible-emoji": "warn",
+    "jsx-a11y/anchor-is-valid": "warn",
+    "jsx-a11y/alt-text": "warn",
 
-    // standard.js
-    "space-before-function-paren": [
+    // prettier - gatsby-starter-default options
+    "prettier/prettier": [
       "error",
       {
-        named: "always",
-        anonymous: "always",
-        asyncArrow: "always",
+        printWidth: 80,
+        tabWidth: "error",
```

Comment:
I think this should be something like this
```suggestion
        tabWidth: 2,
```
otherwise it fails

Reply:
oh wow good catch. I replaced all 2's with "error" to make it more readable to newcomers, but I did not even catch that.